airlock/templates/file_browser/_includes/directory_table/request.html

```diff
@@ -24,6 +29,13 @@
         <td>{{ path.request_filetype.name|title }}</td>
         <td data-order="{{ path.size }}">{{ path.size_mb }}</td>
         <td>{{ path.modified_at|date:"Y-m-d H:i" }}</td>
+        {% if multiselect_withdraw %}
+          <td>
+            {% if True %}
```

Comment:
why `if True`?

Reply:
:facepalm: Leftover debugging! Thanks for catching